users/templates/base.html

```diff
@@ -37,39 +38,39 @@
         <ul class="list-unstyled components">
             <li>
                 {% if request.path == home_url %}
-                <a class="active" href="{% url 'home' %}">
+                <a class="active" href="{{ home_url }}">
                     <img src="{% static 'users/images/white/iko-02.png' %}">
                     <span>{% trans 'Home' %}</span>
                 </a>
                 {% else %}
-                <a href="{% url 'home' %}">
+                <a href="{{ home_url }}">
                     <img src="{% static 'users/images/grey/ico-02.png' %}">
                     <span>{% trans 'Home' %}</span>
                 </a>
                 {% endif %}
             </li>
             <li>
-                {% if request.path == account_url %}
-                <a class="active" href="{% url 'custom-user-update' %}">
+                {% if request.path == account_url or request.path|startswith:'/users/accounts/password_change/'%}
```

Comment:
Is `request.path|startswith:account_url` working? If yes please use this instead of hardcoded url that might change.

Reply:
With `account_url` it didn't work but I have used `password_change url`

Reply:
@Szymiks I think you should then use those reversed urls instead of hardcoded ones, but I see you have left 3 of them. Would it be possible to fix them in separate pull request?